client/src/pages/Signup.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Signup = ({AuthModal}) => {
+    return (
+        <>
+            {AuthModal}
+        </>
```

Comment:
Same as **client/src/pages/Signin.js**. In fact, other than the name of the component, the files are exactly identical. 
Are they going to diverge in the future? Because if they are not, maybe there's an opportunity to eliminate some repeated code.